### PR TITLE
WRP-3074: Update package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated package.json dependencies to fix security vulnerabilities.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3074

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
...
   "devDependencies": {
     "@enact/docs-utils": "^0.4.0",
-    "@enact/ui-test-utils": "^1.0.1",
+    "@enact/ui-test-utils": "^1.0.2",
     "eslint-config-enact-proxy": "^1.0.5",
     "ilib": "^14.15.1"
   }
-}
\ No newline at end of file
+}
```
Comment (daniel-stoian-lgp) [inline package.json:66]: can you remove this added line? :)

**Review by alexandrumorariu — COMMENTED** [review]
Please update this dependency: 
Agate: immer  ^9.0.17  →  ^9.0.18

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**